### PR TITLE
Support docker-compose v2 in install

### LINKED ISSUE #598: [BUG] Install script doesn't support docker-compose v2
**Describe the bug**

The install script assumes the user has docker-compose v1 installed or docker-compose v2 with compose-switch installed.

**To Reproduce**
1. Install `docker-compose` v2 (https://github.com/docker/compose#linux)
2. Ensure compose-switch is NOT installed
3. Run install script
4. Script warns docker-compose is not installed
5. Script attempts to run docker-compose commands

**Expected behavior**
We should support compose v1 and v2 if possible

I don't think `which` (as is currently used) or `command -v` (which we should probably use) will support figuring out if we should use `docker-compose` or `docker compose`.  If there is no `docker-compose` command, the only way to check for v2 *might* be to check if the file `docker-compose` exists in any of the installation locations.

### PR DESCRIPTION
**Pull request by stumpylog**
## Proposed change

This fixes the install script so it will support docker-compose v2, which updated from being a standalone command to a sub-command/plugin of the main docker command.

The script now checks if `docker-compose` command exists, then checks for `docker compose`.  If one of those is found, it's used in following steps.

Fixes #598

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please explain)

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [x] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [x] If applicable, I have checked that all tests pass, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#back-end-development).
- [x] I have run all `pre-commit` hooks, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#code-formatting-with-pre-commit-hooks).
- [x] I have made corresponding changes to the documentation as needed.
- [x] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: install-paperless-ngx.sh [hunk 1]
```diff
@@ -47,24 +47,29 @@ if [[ $(id -u) == "0" ]] ; then
...
-	echo "docker-compose executable not found. Is docker-compose installed?"
-	exit 1
+DOCKER_COMPOSE_CMD="docker-compose"
+if ! command -v ${DOCKER_COMPOSE_CMD} ; then
+	{
+		docker compose version &> /dev/null && DOCKER_COMPOSE_CMD="docker compose"
+		} || {
+		echo "docker-compose executable not found. Is docker-compose installed?"
+		exit 1
+	}
```
Comment (tribut) [inline install-paperless-ngx.sh:67]: This is a bit hard to read, I think. How about just using two "normal" `if`s?

```bash
if ! command -v ${DOCKER_COMPOSE_CMD} ; then
  if docker compose version &> /dev/null ; then
    DOCKER_COMPOSE_CMD="docker compose"
  else
    echo "docker-compose executable not found. Is docker-compose installed?"
    exit 1
  fi
fi
```

**Review by tribut — COMMENTED** [review]
Looks good, thanks!

I have one minor style issue, what do you think?

File: install-paperless-ngx.sh (hunk 1 above)
Comment (stumpylog) [reply install-paperless-ngx.sh]: I'm not really sure why I didn't go for that in the first place.  Updated now

**Review by tribut — APPROVED** [review]
LGTM 👍

**Pull request merged**